### PR TITLE
JWWEB-703: Add backup action to the "Create Wallet" process

### PR DESCRIPTION
**Pull request by stcherenkov**
Also:
- Simplify wallet creation for both Create and Import
- Move copy logic to CopyableField
- Update Google Analytics events

### CODE REVIEW

File: flow-typed/redux/wallets.js [hunk 1]
```diff
@@ -130,23 +130,14 @@ declare type WalletsState = {|
   +password: string,
   +passwordHint: string,
   +passwordConfirm: string,
+  mnemonic: string,
```
Comment (IViol) [inline flow-typed/redux/wallets.js]: `+`

File: src/components/CopyableField/CopyableField.js [hunk 2]
```diff
@@ -4,18 +4,21 @@ import React, { PureComponent } from 'react'
...
+} from 'services'
 
 type Props = {|
-  +copy: CopyableFieldHandler,
-  +download: ?CopyableFieldHandler,
+  +onCopySuccess: Function,
+  +onDownloadSuccess: Function,
   +value: string,
   +valueToDisplay: ?string,
+  isDownloadAvailable: boolean,
```
Comment (IViol) [inline src/components/CopyableField/CopyableField.js]: `+`

File: flow-typed/redux/wallets.js (hunk 1 above)
Comment (stcherenkov) [reply flow-typed/redux/wallets.js]: Fixed, thanks

File: src/components/CopyableField/CopyableField.js (hunk 2 above)
Comment (stcherenkov) [reply src/components/CopyableField/CopyableField.js]: Fixed, thanks

**Pull request merged**